test/Psr7ServerRequestTest.php

```diff
@@ -247,19 +248,36 @@ public function getResponseData()
                         'test1' => [
                             'name' => 'test1.txt',
                             'type' => 'text/plain',
-                            'tmp_name' => '/tmp/phpXXX',
+                            'tmp_name' => __DIR__.'/Psr7ResponseTest.php',
```

Comment:
I had to change this because impossible to get stream metadata on not existing file